scripts/create-kubestellar-demo-env.sh

```diff
@@ -70,16 +78,27 @@ context_clean_up() {
     done <<< "$output"
 }
 
+# Run context cleanup
 context_clean_up
 echo "Context space clean up completed"
 
+# Install KubeStellar core
 echo -e "\nStarting the process to install KubeStellar core: kind-kubeflex..."
-export KUBESTELLAR_VERSION=0.23.1
```

Comment:
BTW, I had forgotten that this script even exists. That is why this line is not mentioned in the release process document, which lists (or rather, is supposed to list) all the places where the release version number sits.